src/pages/DiagramsPage.vue

```diff
@@ -158,7 +159,8 @@ function zoom(plus) {
 
 /**
  * Pan on diagrams container according on the delta.
- * @param {Object} delta - Delta of distance (in pixels) since handler was called last time.
+ * @param delta.delta
```

Comment:
rename this + add type and explanation

Reply:
example 
![image](https://github.com/ditrit/leto-modelizer/assets/62500081/2c126b44-9dfa-484a-8002-0d8ca93cc0a7)

Reply:
done